charts/zenko/values.yaml

```diff
@@ -95,4 +95,17 @@ redis-ha:
     create: true
   replicas:
     servers: *nodeCount
-    sentinels: *nodeCount
\ No newline at end of file
+    sentinels: *nodeCount
+
+grafana:
+  sidecar:
+    datasources:
+      enabled: true
+      # Every config map with the following label will be used as datasource. By default,
+      # promtheus is set as datasource under zenko/templates/datasources.yaml
+      label: grafana-datasource
+    dashboards:
+      enabled: true
+      # Likewise, every config map with the following label will be used as a source for
+      # a dashboard.
+      label: grafana-dashboard
```

Comment:
Guess this will become a tricky one again cfr. comment above... Maybe the `tpl` function can be of use.

Reply:
@NicolasT How exactly can this value be templated to include the release name? Grafana doesn't use partials to set this value so it cannot be overwritten through a define in ```_helpers.tpl``` as far as I know.

Reply:
Good point. One of the shortcomings of Helm when it comes to chart composition.